### PR TITLE
Bugfix/random pep

### PR DESCRIPTION
**Pull request by acepace**
# What is this? 
Doesn't fix anything but brings the number of flake8 warnings down to below 100.

### CODE REVIEW

File: monkey/infection_monkey/exploit/win_ms08_067.py [hunk 1]
```diff
@@ -241,8 +241,8 @@ def _exploit_host(self):
                       build_monkey_commandline(self.host, get_monkey_depth() - 1)
 
         try:
-            sock.send("start %s\r\n" % (cmdline,))
-            sock.send("net user %s /delete\r\n" % (self._config.user_to_add,))
```
Comment (ShayNehmad) [inline monkey/infection_monkey/exploit/win_ms08_067.py:245]: This change will require testing

Comment (acepace) [reply monkey/infection_monkey/exploit/win_ms08_067.py:245]: that it's binary? I did a standalone test. socket.send indeed receives binary digits and not string. Or what sort of test did you have in mind?

Comment (ShayNehmad) [reply monkey/infection_monkey/exploit/win_ms08_067.py:245]: Check the exploiter still works

Comment (VakarisZ) [reply monkey/infection_monkey/exploit/win_ms08_067.py:245]: Problematic, MonkeyZoo doesn't have a test for this

**Comment by acepace** [comment]
@ShayNehmad force pushed the branch and removed the MS08_67 code changes.
Please merge.

**Pull request merged**